ui/js/controllers/logviewer.js

```diff
@@ -229,7 +251,15 @@ logViewerApp.controller('LogviewerCtrl', [
                                 });
                             } else {
                                 $timeout(function() {
-                                    angular.element('.lv-error-line').first().trigger('click');
+                                    if (isNaN($scope.selectedBegin)) {
+                                        angular.element('.lv-error-line').first().trigger('click');
+                                    }
+                                    else {
+                                        var steps = $scope.artifact.step_data.steps;
+                                        var line = $scope.selectedBegin - 8;
+                                        $scope.displayedStep = steps[2];
+                                        $scope.scrollTo($event, steps[2], line);
```

Comment:
So I guess $event here and on L238 is what is triggering the console error when loading a hash logviewer URL
eg. *.../logviewer.html#?job_id=11038129&repo=mozilla-inbound#L1399*

I spent a little bit checking to see if it was a timing thing on page load, and it doesn't seem to be. I'll defer to @camd on this when he gets to rest of the review.